src/vos/vos_layout.h

```diff
@@ -245,12 +245,14 @@ struct vos_cont_df {
 
 /** btree (d/a-key) record bit flags */
 enum vos_krec_bf {
-	/* The record has an evtree */
+	/* Array value (evtree) */
 	KREC_BF_EVT			= (1 << 0),
+	/* Single Value or Key (btree) */
+	KREC_BF_BTR			= (1 << 1),
```

Comment:
Each dkey has an akey subtree and each akey has an sgv subtree.   Both share the same btree and both have the flag set.   So it would need to be SGV_OR_AKEY or some such to be less generic.